TuneUp/ProfiledNodeViewModel.cs

```diff
@@ -130,6 +132,11 @@ public string StateDescription
             }
         }
 
+        /// <summary>
+        /// The Stopwatch to measure execution time of this node
+        /// </summary>
+        public Stopwatch Stopwatch { get; set; }
```

Comment:
Do we need this as public? Can we make this internal?

Reply:
Good point. It's internal now.